### DISCUSSION TITLE
Display Child Popup from a Parent Popup

### DISCUSSION THREAD

**Discussion by Orgbrat** [CommunityToolkit/Maui#332] [Q&A] [answered]
So is it possible to show a second popup from the original popup already displayed?

**Comment by pictos** [comment]
@Orgbrat I'm not sure, I remember an issue on XCT about it, looks like you can do it on Android but not on iOS. I didn't take a look to see if the limitation is on the platform or on us. If you tried and have some results, feel free to share <3

**Reply by Orgbrat** [reply]
@pictos Thanks for the quick reply. I will try to show a Popup from a Popup and share my results.

**Comment by Orgbrat** [comment, marked as answer]
@pictos  Doesn't work in Windows either. You have to close the parent and then the child opens. I needed the child to open on top of the parent and then when the child closed it transferred data to the parent. Oh well I will have to rethink my process.

**Reply by pictos** [reply]
Can you open a discussion asking for this feature? I can take a look in the near future. Looks like you aren't the only one that need this feature https://github.com/xamarin/XamarinCommunityToolkit/issues/1764

**Reply by Orgbrat** [reply]
New Feature discussion [Need ability to open second level popup (e.g. to open a child popup from the parent popup) #335](https://github.com/CommunityToolkit/Maui/discussions/335) created.

**Reply by pictos** [reply]
Thanks, @Orgbrat, you rock❣️

**Comment by CliffAgius** [comment]
Closed as Answered.
